lib/apollo/hooks/state/meActivity/meActivity.ts

```diff
@@ -0,0 +1,25 @@
+import { filterAvailableNodes } from 'helpers';
+import { useMeActivitiesQuery } from 'graphql/queries/__generated__/meActivities.generated';
+import { Activity, ActivityEvent } from 'graphql/types';
+import { TActivity } from './types';
+
+export const useMeActivities = ({ before, after, event, pageSize = 5 }: TActivity) => {
+  const { loading, error, data } = useMeActivitiesQuery({
+    variables: {
+      events: event ? [event] : Object.values(ActivityEvent),
```

Comment:
Why we don't use `events` in the `useActivities`?

Reply:
It is different query, public and private. With different signature. 
![image](https://github.com/fs/react-ts-next-base/assets/57991929/56739022-3e1c-4b7b-ad25-8b570f885192)
![image](https://github.com/fs/react-ts-next-base/assets/57991929/de85c074-e6ce-4feb-b8e5-287acc8dc9df)